package.json

```diff
@@ -4,6 +4,7 @@
     "packages/*"
   ],
   "scripts": {
+    "precommit": "node localizeLanguage.js && lint-staged && flow",
```

Comment:
Since we upgraded Husky, there is no longer any such thing as a "precommit" script. It's now in a different part of package.json. You might have to run yarn to get the latest stuff installed if it isn't working.